include/swift/AST/RuntimeVersions.def

```diff
@@ -160,7 +160,10 @@ RUNTIME_VERSION(
 
 RUNTIME_VERSION(
   (6, 2),
-  FUTURE
+  PLATFORM(macOS,   (26, 0, 0))
+  PLATFORM(iOS,     (26, 0, 0))
+  PLATFORM(watchOS, (26, 0, 0))
+  PLATFORM(visionOS,(26, 0, 0))
```

Comment:
tvOS?

Reply:
tvOS inherits directly from iOS, so its versions don't get specified anywhere in this file